python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -231,6 +249,79 @@ def run(self, splash=None, version=None):
         result = app.exec_()
         return result
 
+    def uses_legacy_authentication(self):
+        """
+        Returns if the Desktop's bootrap code uses the tk-framework-login for
```

Comment:
IMHO "Desktop" in this context is confusing because it can refer to both the tk-desktop engine and the installed desktop. Maybe be explicit and say "Shotgun Desktop's installed code" or something?